src/chunk_scan.c

```diff
@@ -240,6 +253,8 @@ ts_chunk_scan_by_constraints(const Hyperspace *hs, const List *dimension_vecs,
 				unlocked_chunks[unlocked_chunk_count] = chunk;
 				unlocked_chunk_count++;
 
+				if (ts_chunk_is_osm_chunk(chunk))
```

Comment:
If the OSM chunk doesn't have dimension slices, should we `Assert(!ts_chunk_is_osm_chunk(chunk))` here?

Reply:
the chunk has dimension slices, but are dummy ranges.

Reply:
Is it ever going to appear in the results of a lookup by constraints? I guess not, so probably we could assert this.

Reply:
I mean, if it did appear there sometimes, it would be weird -- when, and why not always?

Reply:
Yes, it should appear in this list with a query that has < operator:
e.g. select * from ht where ts < '2020-01-01'